lib/Ride.dart

```diff
@@ -182,37 +164,38 @@ class _RideCardState extends State<RideCard> {
                       Container(
                         decoration: BoxDecoration(
                           shape: BoxShape.circle,
-                          boxShadow: [dropShadow],
+                          boxShadow: [CarriageTheme.shadow],
                         ),
                         child: CircleAvatar(
                           radius: 24,
                           //TODO: replace with rider's image
                           backgroundImage:
-                              AssetImage('assets/images/terry.jpg'),
+                          AssetImage('assets/images/terry.jpg'),
```

Comment:
Nit: Any reason why the indentation was removed?

Reply:
I think this was an accident sorry! Reminds me, some of our indentation is really messed up from passing around between Android Studio and VSCode which seems to mess up end parentheses--maybe would be a simple code cleanup task?

Reply:
Definitely may be a good cleanup task